addons/l10n_es_website_sale/controllers/main.py

```diff
@@ -0,0 +1,43 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.addons.website_sale.controllers.main import WebsiteSale
+
+
+class L10nESWebsiteSale(WebsiteSale):
+
+    def _validate_address_values(self, address_values, partner_sudo, address_type, use_delivery_as_billing, *args, **kwargs):
+        invalid_fields, missing_fields, error_messages = super()._validate_address_values(
+            address_values, partner_sudo, address_type, use_delivery_as_billing, *args, **kwargs
+        )
+
+        return invalid_fields, missing_fields, error_messages
```

Comment:
Why do we need this ? We just return the `super()`. Am I missing something ?

Reply:
We don't need this, just forgot to remove it ... You didn't see anything :eyes: